.github/workflows/audit-check.yml

```diff
@@ -0,0 +1,24 @@
+name: Check dependencies against Advisory DB
+on:
+  push:
+    paths:
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+  pull_request:
+    paths:
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+jobs:
+  audit:
+    name: Security audit
+    env:
+      CARGO_TERM_COLOR: always
```

Comment:
only used here, and not in audit-daily?

Reply:
daily should create an issue if there is a violation so I was hoping that nobody would ever look into the CLI output of it. Don't have a strong opinion about it though....

Reply:
ah that makes sense, as long as all relevant info is captured in the issue